jose.py

```diff
@@ -35,13 +35,15 @@
     'br?',
     'u died [real] [Not ClickBait]',
     'rEEEEEEEEEEEEE',
+    'you didn\'t buy gold',
```

Comment:
When contributing to José, please obey standard PEP8 conventions.

According to the [string quotes section of PEP8](http://pep8.org/#string-quotes), you should use the other type of string quote token (in this case, `"`) in order to avoid escapes.

Reply:
I DONT CARE
FUCK STANDARDS

Adryd fucks standards [2018] [real] [not clickbait] 

okay ill fix

Reply:
@lnmds fix your code?

Reply:
actually I'll send a pr too why not lol